client/test/e2e/enterprises/enterprises.spec.js

```diff
@@ -0,0 +1,83 @@
+/*global describe, it, element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var FormUtils = require('../shared/FormUtils');
+
+describe('The Enterprises Module', function () {
+
+  // shared methods
+  var path = '#/enterprises';
+  var ENTERPRISES = {
```

Comment:
Since this is a single JSON, it would be better to use the singular form "ENTERPRISE" instead of the plural "ENTERPRISES".